model-optimizer/extensions/ops/ONNXResize11.py

```diff
@@ -61,12 +61,14 @@ def onnx_resize_infer(node: Node):
             assert input2_value is not None, \
                 "Node {} with op {} has no value in input port 2".format(node.soft_get('name', node.id), node.op)
             scale = np.array(input2_value)
-            output_shape = np.floor(input_shape * scale + 1.0e-6).astype(np.int64)
+            output_shape = np.floor((input_shape + 1.0e-6) * scale).astype(np.int64)
```

Comment:
Why did you change this? Have you seen such a formula in the ONNX runtime?

Reply:
Fixed.